exercise-solutions/files-match-result/src/main.rs

```diff
@@ -0,0 +1,14 @@
+use std::fs::File;
+
+
+fn main() {
+    let _open_result = File::open("src/lib/content.txt").unwrap();
+                    // ^^^^^^^^ File::open yields a Result<File, Error>, 
```

Comment:
Watch out. If you `cargo fmt` this file the comment will move back to the left.
You have to put the whitespace after the `//` to keep the `^^^` where you want it.

Reply:
Maybe this should should say:

```rust
fn main() {
    println!("This is the solution for the files-match-result exercise.");
    println!("Run like 'cargo run --example step_3' to run the Step 3 solution");
}
```